src/procedures/agm_45.md

```diff
@@ -1,5 +1,10 @@
 # AGM-45 Missile
 
+The checklist below shows the necessary actions that are needed to be taken inside the cockpit. 
+In order to have a better understanding of the AGM-45 Shrike missile employment methods 
+please refer to the sections [4.3.3.1 AGM-45 Shrike](http://f4.manuals.heatblur.se/stores/air_to_ground/missiles/shrike.html#agm-45-shrike-anti-radiation-missile)
+and [4.3.3.6.5 Lofting and Tossing](https://f4.manuals.heatblur.se/stores/air_to_ground/bombs/employment/lofting_tossing.html)
```

Comment:
good stuff. these links should be internal/relative links though and not external.
so like `[foo](../../foo/bar)` and similar.
i can also take care of it on monday or sth